src/generate/filterExtensionDefinitions.ts

```diff
@@ -0,0 +1,19 @@
+import { DocumentNode, DefinitionNode, Kind } from 'graphql';
+
+export default function filterExtensionDefinitions(ast: DocumentNode) {
+  const extensionDefs = ast.definitions.filter(
```

Comment:
`graphql-js` actually provides us this exact filter predicate via [`isTypeSystemExtensionNode`](https://github.com/graphql/graphql-js/blob/5c9946b0d0e2e082772db4ccd50013caa0b1ed79/src/language/predicates.js#L70), can we use that here instead?

Reply:
I considered that, but that predicate wasn't introduced until, I think, 14.0.0.  I do think that any version of `graphql-tools` that has narrowed its `peerDependencies` to exclude pre-14.x should be using that predicate though!